src/cargo/ops/cargo_add/dependency.rs

```diff
@@ -822,6 +921,47 @@ impl std::fmt::Display for GitSource {
     }
 }
 
+/// Dependency from a workspace
+#[derive(Debug, Hash, PartialEq, Eq, Clone)]
+#[non_exhaustive]
+pub struct WorkspaceSource {
+    /// if workspace = true or false
+    pub workspace: bool,
```

Comment:
Can this ever be false?  Should we just have the presence of this assume `true`?

Reply:
That is up to you. I don't think it should be false ever since we check if it's false before we create it.

Reply:
Let's just have this be a unit struct for now.  We can always add fields later